themes/vocabulary_theme/templates/blocks/get-involved.html

```diff
@@ -1,19 +1,22 @@
-<section class="get-involved">
+<section class="get-involved" aria-labelledby="get-involved-title">
```

Comment:
`sections` should automatically attach to the nested `h2` semantically, this `aria-labelledby` shouldn't be necessary.